### PR TITLE
Add note about grading before course end date

### PR DESCRIPTION
**Pull request by srpearce**
## [DOC-3186](https://openedx.atlassian.net/browse/DOC-3186)

Add brief note to [Specify the Minimum Credit-Eligible Grade](http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/course_features/credit_courses/credit_courses.html#specify-the-minimum-credit-eligible-grade) instructing course teams to make sure grading is complete before the course end date.
### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [ ] Doc team review (sanity check): @edx/doc
- [ ] Product review: Lauren Holliday
### Testing
- [ ] Ran ./run_tests.sh without warnings or errors
### Post-review
- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: en_us/course_authors/source/course_features/credit_courses/credit_courses.rst [hunk 1]
```diff
@@ -33,6 +33,10 @@ steps.
 Specify the Minimum Credit-Eligible Grade
 ********************************************
 
+.. note::
+  To make sure that learners' grades are correct, grading for credit courses
+  must be complete before the course end date.
+
```
Comment (lamagnifica) [inline en_us/course_authors/source/course_features/credit_courses/credit_courses.rst:41]: maybe xref to where you get info on how to "Complete Grading"?

**Comment by lamagnifica** [comment]
@srpearce I'm not sure exactly what they're supposed to do. The [Assign Grades](http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/student_progress/checking_student_progress.html) section seems to indicate that the course has to be over before you run the final grade report. Maybe update both this file and that assign grades section with a different procedure/advice for credit vs. non-credit courses?

**Comment by srpearce** [comment]
Meeting 9/28 to discuss grading (see also ECOM-5298)

**Comment by srpearce** [comment]
@edx/doc This is ready for another review. I made some changes to the checking_student_progress.rst file for organization/clarity.

@Lauren Holliday I added the following note to clarify the assigning grades/issuing certificates timeline to credit_courses.rst:

To make sure that learners' grades are correct, you must assign final grades for credit courses before you create certificates for the course. You assign final grades after the course end date. For more information, see :ref:`Checking Student Progress and Issuing Certificates`.

File: en_us/shared/manage_live_course/bulk_email.rst [hunk 2]
```diff
@@ -790,8 +790,8 @@ about future access to course materials. Be sure to replace values enclosed by
 
   * If you qualify for a certificate (overall score {number}% or higher), the
     edX dashboard will include a link to your certificate in the near future.
-    While you may see the link in a few days, it can take up to two weeks edX
-    to generate all of the course certificates.
+    While you may see the link in a few days, it can take up to two weeks for
+    edX to generate all of the course certificates.
```
Comment (lamagnifica) [inline en_us/shared/manage_live_course/bulk_email.rst:795]: Is this still true? Does edX still generate certificates?

File: en_us/shared/student_progress/checking_student_progress.rst [hunk 3]
```diff
@@ -18,79 +17,104 @@ For more information, see the following other topics about certificates.
 * :ref:`Reporting Certificate Data`
 
 ****************************************
-Overview
+Send Farewell Message
```
Comment (lamagnifica) [inline en_us/shared/student_progress/checking_student_progress.rst]: Adding these headings is great! These topics are more conceptual, and not so much procedural though, so I think they would get gerunds as headings.

File: en_us/shared/student_progress/checking_student_progress.rst [hunk 4]
```diff
@@ -18,79 +17,104 @@ For more information, see the following other topics about certificates.
...
+******************
 
-If you have a self-paced course, you can allow your learners to request their
-certificates as soon as they have completed enough of the course with a high
-enough grade to qualify for a certificate. Learners might want to request a
-certificate right away, or they might wait until they have completed the
-course.
+Typically, you generate and issue certificates after the course ends. However,
+for self-paced courses, you can allow your learners to request and download
+on-demand certificates as soon as they have completed enough of the course
```
Comment (lamagnifica) [inline en_us/shared/student_progress/checking_student_progress.rst]: this is a little hard to parse. Maybe ...enough of the course ^and with a high enough grade... ?

File: en_us/shared/student_progress/checking_student_progress.rst [hunk 5]
```diff
@@ -18,79 +17,104 @@ For more information, see the following other topics about certificates.
...
+with a high enough grade to qualify for a certificate.
 
-========================================
-Allow Learners to Request Certificates
-========================================
 
-To allow learners to request certificates from the **Progress** page, follow
-these steps.
+===================================================================
+Generate Certificates for Instructor-Paced and Self-Paced Courses
```
Comment (lamagnifica) [inline en_us/shared/student_progress/checking_student_progress.rst]: and --> or (?) 
It seems like there is another option available for self-paced, so "and" might be misleading

File: en_us/shared/student_progress/checking_student_progress.rst [hunk 6]
```diff
@@ -18,79 +17,104 @@ For more information, see the following other topics about certificates.
...
+   To generate certificates for your course, work with your edX partner
+   manager.
 
-#. Select **Enable Student-Generated Certificates**.
+.. only:: Open_edX
 
-   When they have qualified, learners can request their certificates from the
-   **Progress** page.
+   To generate certificates for your course, work with the administrator of
+   your instance of Open edX.
```
Comment (lamagnifica) [inline en_us/shared/student_progress/checking_student_progress.rst]: can we point them to instructions, or add instructions? we've gotten many requests (and a couple of PRs from the community) to add "admin" steps right in to the B&R guide, or to xref to the OCR guide, so that they can find out what specifically needs to get done.

File: en_us/shared/student_progress/checking_student_progress.rst [hunk 7]
```diff
@@ -18,79 +17,104 @@ For more information, see the following other topics about certificates.
...
```
Comment (lamagnifica) [inline en_us/shared/student_progress/checking_student_progress.rst:66]: I'm finding this a bit confusing. Is there a sequence of steps they need to go through, only one of which is the generation phase (which they have no control over)? what should they do before this generation phase, and what could they do after/in parallel? Maybe a use case or two would make the options and the workflow clearer.

File: en_us/shared/student_progress/checking_student_progress.rst [hunk 8]
```diff
@@ -18,79 +17,104 @@ For more information, see the following other topics about certificates.
...
 
-   To prevent learners from requesting certificates, select **Disable Student-
-   Generated Certificates**.
+====================================================
+Enable On-Demand Certificates for Self-Paced Courses
+====================================================
+
+If you want your learners to be able to receive their certificates as soon as
+they qualify for a certificate, you must change an LMS setting to allow
```
Comment (lamagnifica) [inline en_us/shared/student_progress/checking_student_progress.rst]: maybe break this into two sentences?
You can set up your course so that learners can download their certs as soon as they qualify.
To do so, change...

maybe the contents directive would help here

File: en_us/shared/student_progress/checking_student_progress.rst [hunk 9]
```diff
@@ -18,79 +17,104 @@ For more information, see the following other topics about certificates.
...
 
-==============================================================
-Allow Learners to Download Certificates when They Qualify
-==============================================================
+#. Select **Enable Student-Generated Certificates**.
 
-To allow learners to download certificates from the dashboard when they qualify
-for them, you must modify the **Certificates Display Behavior** advanced
-setting in Studio.
+   When they have qualified, learners can request their certificates from the
```
Comment (lamagnifica) [inline en_us/shared/student_progress/checking_student_progress.rst:108]: ^After you change this setting, learners who have qualified can...

do they "request" or "download" certs? request seems odd to me... I've read through a few times and can't quite tell if you can 1) change either the LMS setting or both the LMS and the Studio setting, or 2) must change both (and if so, what happens if you forget and only do one?) An xref to the SFD that describes what learners see would be useful too I think.

Is there anything course teams should know about changing this setting in a course that is already running? Would it be worth adding an xref to this section from the section where we describe setting up a self-paced course?

File: en_us/shared/student_progress/checking_student_progress.rst [hunk 10]
```diff
@@ -18,79 +17,104 @@ For more information, see the following other topics about certificates.
...
+   **Progress** page.
+
+   To prevent learners from requesting certificates, select **Disable Student-
```
Comment (lamagnifica) [inline en_us/shared/student_progress/checking_student_progress.rst]: again, "request" here seems odd

**Review by lamagnifica — COMMENTED** [review]
I'm a bit confused by the expected workflow and what the alternatives are.
